ecosystem/indexer-grpc/indexer-grpc-fullnode/src/convert.rs

```diff
@@ -760,6 +760,9 @@ pub fn convert_transaction(
         Transaction::ValidatorTransaction(_) => {
             transaction::transaction::TransactionType::Validator
         },
+        Transaction::BlockMetadataExtTransaction(_) => {
+            todo!()
```

Comment:
These will be needed for indexing I think?

Reply:
yes, @bowenyang007 @larry-aptos i assume you guys will have a sub-PR for here?

Reply:
n/a